packages/api/src/readDataFeeds.ts

```diff
@@ -8,16 +8,25 @@ const CONFIG_URL = process.env.TEST_BRANCH
   ? `https://raw.github.com/witnet/data-feeds-explorer/${process.env.TEST_BRANCH}/packages/api/src/dataFeedsRouter.json`
   : 'https://raw.github.com/witnet/data-feeds-explorer/main/packages/api/src/dataFeedsRouter.json'
 
+function isRouterDataFeedsConfig (val: any): val is RouterDataFeedsConfig {
+  return val?.abi && val?.chains
+}
+
 export async function fetchDataFeedsRouterConfig (): Promise<
   RouterDataFeedsConfig | null
 > {
   return await axios
     .get(CONFIG_URL)
     .then(res => {
-      return res.data
+      if (isRouterDataFeedsConfig(res.data)) {
+        return res.data
+      } else {
+        throw new Error('Received data is not of type RouterDataFeedsConfig')
+      }
     })
     .catch(err => {
-      console.log('There was an error fetching the config file', err)
+      console.error('There was an error fetching the config file', err)
```

Comment:
This may better help spotting misconfigured URLs:
```suggestion
      console.error(`There was an error fetching the config file from {CONFIG_URL}`, err)
```